developer_manual/app/tutorial/javascript_and_css.rst

```diff
@@ -31,3 +31,27 @@ As with ``script``, the first parameter is the application to find the CSS file
 .. code-block:: php
    
    style('ownnotes, 'style');  // adds ownnotes/css/style.css
+   
+Asset Management
```

Comment:
I'd fully drop this - every framework and tool set has it's own way of doing this.
e.g. webpack is the latest toolset to be used.

@PVince81 @felixheidecke please comment - THX

Reply:
Sounds fair. And frontend tools and frameworks change almost every other day. :-D

Reply:
@PVince81 @felixheidecke, your thoughts?